srcs/server/socket.cpp

```diff
@@ -115,16 +119,24 @@ namespace ft
 
 	Socket::RecievedMsg Socket::recieve_msg()
 	{
-		std::cout << "poll_fd_vec_.size(): " << poll_fd_vec_.size() << std::endl;
-		for (size_t i = 0; i < poll_fd_vec_.size(); ++i) {
-			std::cout << poll_fd_vec_[i].fd << " e" << poll_fd_vec_[i].events << " re" << poll_fd_vec_[i].revents;
-			std::cout << (i < poll_fd_vec_.size() - 1 ? " : " : "");
-		}
-		std::cout << std::endl;
+		
+		print_event_debug();
 
 		int poll_rslt = poll(&poll_fd_vec_[0], poll_fd_vec_.size(), 1000);	
-		if (poll_rslt == -1)
-			throw SetUpFailException("Error: poll()");
```

Comment:
If poll has an error, in general we will just continue the loop
if EINVAL(maybe too many fds) or ENOMEM, remove the last fd of the list and continue the loop